src/settings/FinePolicy/components/EditSections/OverdueAboutSection/OverdueAboutSection.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import '../../../../../../test/jest/__mock__';
+
+import OverdueAboutSection from './OverdueAboutSection';
+
+const mockedName = 'John Doe';
+const mockedDescription = 'My description!';
+
+jest.mock('react-final-form', () => ({
```

Comment:
This mock should be replaced in `test/jest/__mock__/` folder, create new file for final-form components since you will need them to use in another components

Reply:
done